lib/ci-stack.ts

```diff
@@ -52,6 +52,8 @@ export interface CIStackProps extends StackProps {
   readonly enableViews?: boolean;
   /** Use Production Agents */
   readonly useProdAgents?: boolean;
+  /** Specify jenkins instance type */
+  readonly jenkinsType?: string;
```

Comment:
Nit: `jenkinsDeploymentType`?

Reply:
Or jenkinsInstanceType, Which name would you recommend?

Reply:
I'm good with both.